peachjam/views/generic_views.py

```diff
@@ -75,12 +75,14 @@ def get_context_data(self, **kwargs):
                 ).values_list("date__year", flat=True)
             )
         )
+        labels = {"Author": "Regional body"}
```

Comment:
The point of this change is that agp will use `Regional body` but other sites like lawlibrary will use `Author`. So this won't work.